src/utils/dom.js

```diff
@@ -61,5 +61,5 @@ export function $(e, parent) {
             ? // a single element is wrapped in an array
               [e]
             : // selector and NodeList are converted to Element[]
-              [].slice.call(e[0].nodeName ? e : (parent || root).querySelectorAll(e));
+              [].slice.call(e[0].nodeName ? e : document.querySelectorAll(e));
```

Comment:
How does this fix "Document not defined?" This is also a departure from the expected logic that finds the elements inside a parent if one is provided.

Reply:
As far as I can tell, this is the only logic change. If there's something I missed, do let me know. In the future, try to scope PRs to more focused changes so real changes are more apparent.